test/regressions/index.test.js

```diff
@@ -119,6 +113,21 @@ async function main() {
           '[data-testid="testcase"]:not([aria-busy="true"])',
         );
 
+        // Wait for the flags to load
+        await page.waitForFunction(
+          () => {
+            const images = Array.from(document.querySelectorAll('img'));
+            return images.every((img) => {
+              if (!img.complete && img.loading === 'lazy') {
```

Comment:
For lazy-loaded images that are not visible `img.complete` will always be `false`